website/src/libs/api-v3/cron.js

```diff
@@ -181,7 +181,7 @@ export function cron (options = {}) {
     });
     task.completed = false;
 
-    if (completed || scheduleMisses > 0) {
+    if (!completed && scheduleMisses === 0) {
```

Comment:
Hmm. That makes sense. Yea, that makes more sense. I was trying to find this version  of the cron.